lua/gatekeeperc.lua

```diff
@@ -0,0 +1,78 @@
+local ffi = require("ffi")
+
+--Structs
+ffi.cdef[[
+
+struct bp_config {
```

Comment:
I have checked the thread, it seems that it needs to make a patch to the Luajit, and wrap a Lua function "include" for FFI. For now, the Luajit itself doesn't support this functionality. If we decide to use this, I can conduct experiments to test it.